app/views/admin/loans/questionnaires/_breakeven_table_question.html.slim

```diff
@@ -1,28 +1,33 @@
-.table-responsive.editable-tables
+.table-responsive.editable-tables.breakeven-tables
 
   .table-container
     h4.title = t("loan.breakeven.products", count: 2)
 
-    table.table.table-condensed.table-striped.table-bordered.editable-table data-table="products"
+    table.table.table-condensed.table-striped.table-bordered.editable-breakeven-table data-table="products"
```

Comment:
Don't need a separate class here. Just use the fact that it's inside `.breakeven-tables` to select it.